include/onnxruntime/core/graph/graph.h

```diff
@@ -294,17 +294,21 @@ class Node {
   Class to provide const access to Node instances iterated via an EdgeConstIterator. */
   class NodeConstIterator {
    public:
-    NodeConstIterator(EdgeConstIterator p_iter);
+    NodeConstIterator(EdgeConstIterator p_iter) { m_iter = p_iter; }
```

Comment:
Did it make a significant perf impact to pull in the definitions to the header? I am guessing this is done to let the compiler inline those functions.

Reply:
Iterator* and Iterator++ calling was each ~5% in the PriorityNodeCompare function, so in the end it's in the 1% to 2% range. Still, such tiny functions should never be called since the call itself already requires more instructions that the actual instruction itself and calls potentially utilize space in the branch instruction cache affecting other regions of code.